consensus/src/detail/error.rs

```diff
@@ -81,6 +84,8 @@ pub enum BlockError {
     SerializationInvariantError(Id<Block>),
     #[error("Unexpected numeric type conversion error `{0:?}`")]
     InternalNumTypeConversionError(Id<Block>),
+    #[error("Internal block representation is invalid `{0:?}`")]
```

Comment:
I think it's preferable to use `{0}` here rather than `{0:?}`. That would give a proper error message instead of strings in Rust syntax.

Reply:
Done